orchagent/fdborch.cpp

```diff
@@ -440,38 +446,68 @@ void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
 bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const string& type)
```

Comment:
In the FdbOrch::update() function when a MAC is learnt, getPortByBridgePortId might fail for the tunnel bridge
port id as it is not added in the m_portList. Tunnels could be added to the port list or the learn event for tunnel could be handled based on the bridge port type.

Reply:
Yes, getPortByBridgePortId will fail for the tunnel bridge port, FdbOrch::update() function only notify observer. FdbOrch::addFdbEntry() skip observer currently, please see the comment '/*TBD: Vxlan tunnel is not in portlist, here skip observer, refine later*/'. The best way is create tunnel port in m_portlist, this is TBD later.